src/popup/pages/TransactionDetails.vue

```diff
@@ -236,10 +274,15 @@ export default defineComponent({
       isErrorTransaction,
       isAllowance,
       isDex,
+      isMultisig,
```

Comment:
In case of `payForTx` type of transaction fee should be a summary of both fees (inner and external).